cli/bench/deno_http.js

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+async function main(addr) {
+  const rid = await Deno.http.createServer(addr);
+  console.log("Server listening on", addr);
+
+  while (true) {
+    const connRid = await Deno.http.accept(rid);
+    handleConn(connRid);
+  }
+}
+
+async function handleConn(connRid) {
+  while (true) {
+    const [connectionClosed, ...req] = await Deno.http.nextRequest(connRid);
+
+    handle(req);
+
+    if (connectionClosed) {
+      break;
+    }
+  }
+}
+
+async function handle(req) {
+  const [requestBodyRid, responseSenderRid, method, headers, url] = req;
+
+  // Don't care about request body, discard it outright.
+  if (requestBodyRid) {
+    Deno.core.close(requestBodyRid);
+  }
+
+  const resp = await globalThis.handler({ method, headers, url });
+
+  // If response body is Uint8Array it will be sent synchronously
+  // in a single op, in other case a "response body" resource will be
+  // created and we'll be streaming it.
+  const body = resp.body ?? new Uint8Array();
+  const zeroCopyBufs = body instanceof Uint8Array ? [body] : [];
+
+  const responseBodyRid = Deno.http.respond(
+    responseSenderRid,
+    resp,
+    ...zeroCopyBufs,
```

Comment:
avoid using spread operator as arguments length is known to be either 0 or 1 (could also avoid alloc of empty array by propagating it as undefined)

Reply:
Yeah, I had removed this in my local branch. Buffers are no longer variadic so the `...zeroCopyBufs` pattern is discouraged / not useful